src/preferences/dialog/dlgprefreplaygain.cpp

```diff
@@ -3,24 +3,37 @@
 #include "controlobject.h"
 #include "util/math.h"
 
-#define kConfigKey "[ReplayGain]"
-
-static const int kReplayGainReferenceLUFS = -18;
-
-
-DlgPrefReplayGain::DlgPrefReplayGain(QWidget * parent, UserSettingsPointer  _config)
+namespace {
+const char* kConfigKey = "[ReplayGain]";
```

Comment:
We need a dedicated ReplayGainSettings class as I just mentioned. This class 
will be used by both the dialog (read/write) and the analyzers (read only). 
All config keys with their default values should be hidden.

On 03/05/2016 01:39 AM, RJ Ryan wrote:
>
> In src/preferences/dialog/dlgprefreplaygain.cpp 
> <https://github.com/mixxxdj/mixxx/pull/520#discussion_r55109440>:
>
> > @@ -3,24 +3,37 @@
> >  #include "controlobject.h"
> >  #include "util/math.h"
> >
> > -#define kConfigKey "[ReplayGain]"
> > -
> > -static const int kReplayGainReferenceLUFS = -18;
> > -
> > -
> > -DlgPrefReplayGain::DlgPrefReplayGain(QWidget * parent, UserSettingsPointer  _config)
> > +namespace {
> > +const char* kConfigKey = "[ReplayGain]";
>
> Could you move these to a common file and reference them in teh analyzers 
> and here? Too many Analyze vs Analyse differences make it likely to mess 
> up :).
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/520/files#r55109440>.
>